css/layout.css

```diff
@@ -3,11 +3,30 @@
 }
 
 .container {
+  font-family: 'Open Sans', sans-serif;
   max-width: 575px;
   margin: 0 auto;
   padding: 0 1.5rem;
 }
 
+.header-container {
+  font-family: 'Open Sans', sans-serif;
+  max-width: 575px;
+  margin: 0 auto;
+  padding: 0 1.5rem;
+  color: white;
+  background-color: rgb(105, 37, 105);
+}
```

Comment:
Don't make variants of layout classes, you should just be using the `container` class and making an additional class you can add for any styling you need beyond what `container` sets